runtests.py

```diff
@@ -150,10 +150,14 @@ def store_variables_and_writefiles(assignments, source, ns):
         else:
             variables.pop(variable, None)
     
+    if source is None:
+        print("ERROR: expected data, but found None")
+        print("Ignoring file writes")
```

Comment:
Is there any additional context we can provide here regarding implications of this failure?  Perhaps the `assignments` which were skipped?